styles.css

```diff
@@ -0,0 +1,171 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 10px;
+}
+
+/*=====================HEADER - NAV========================*/
+
+h1 {
+    font-size: 2rem;
+    padding: 10px;
+    margin: 0;
+    position: fixed;
+    left: 10px;
+    top: 10px;
+    z-index: 5;
+}
+
+header {
+    height: 500px;
+}
+
+nav {
+    position: relative;
+    height: 350px;
+}
+
+[class|=nav]  {
+    text-align: center;
+    font-size: 1.3rem;
+    color: #ABB2B9;
+    padding-top: 80px;
+    position: absolute;
+    border-radius: 100%;
+    height: 180px;
+    width: 180px;
+    transition: .8s;
+    transition-timing-function: ease-in-out;
+    background-color: #ABB2B9;
+}
+
+[class|=nav] a {
+    text-decoration: none;
+    color: #FFFFFF;
+}
+
+.nav-1 {
```

Comment:
Maybe tag names could be more descriptive? Are there 3 navbars or 3 sections within 1 navbar etc?

Reply:
sorted